exchange/adapter_builders.go

```diff
@@ -204,6 +205,7 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderBlue:              blue.Builder,
 		openrtb_ext.BidderBmtm:              bmtm.Builder,
 		openrtb_ext.BidderBoldwin:           boldwin.Builder,
+		openrtb_ext.BidderBrave:             brave.Builder,
```

Comment:
Thank you for keeping the lists alphabetical. :)

Reply:
for sure, do not want to mess up your clean code